packages/testsuite/cypress/e2e/transaction/transactions-configuration-subsystem-transaction-configuration.cy.ts

```diff
@@ -0,0 +1,177 @@
+// navigation crumbs in the management console
+describe("TESTS: Configuration => Sybsystem => Transaction => Configuration", () => {
+  const transactions = "transactions";
+  const address = ["subsystem", transactions];
+  const configTab = "#tx-attributes-config-item";
+  const configurationForm = "tx-attributes-form";
+  const defaultTimeout = "default-timeout";
+  const enableTsmStatus = "enable-tsm-status";
+  const journalStoreEnableAIO = "journal-store-enable-async-io";
+  const jts = "jts";
+  const maximumTimeout = "maximum-timeout";
+  const nodeIdentifier = "node-identifier";
+  const statisticsEnabled = "statistics-enabled";
+  const useJurnalStore = "use-journal-store";
+
+  let managementEndpoint: string;
+
+  // Here we start our WildFly container prior to test execution
+  before(() => {
+    cy.startWildflyContainer().then((result) => {
+      managementEndpoint = result as string;
+    });
+  });
+
+  after(() => {
+    cy.task("stop:containers");
+  });
+
+  it("Edit Default Timeout", () => {
+    cy.task("execute:cli", {
```

Comment:
could you please create a new method `cy.readAttributeAsNumber` similarly as for `cy.readAttributeAsBoolean`?

Reply:
Should be fixed now, I also added method for writing a attribute